src/main/webapp/i18n/de/aboutUs.json

```
@@ -1,33 +1,106 @@
-                }
-            }
+            "text": "<b>Prüfungsmodus:</b> Dozenten können Online-Prüfungen mit Übungsvarianten, integrierter Plagiatsprüfung, Testläufen und Studierendenbewertungen erstellen. Weitere Informationen finden Sie unter <a href=\"https://artemis.ase.in.tum.de/features/students\">Funktionen für Studierende im Prüfungsmodus</a> und unter <a href=\"https://artemis.ase.in.tum.de/features/instructors\">Funktionen für Dozenten im Prüfungsmodus</a>.",
+            "details": [
+                "Alle Übungen von oben sind auch für Prüfungen verfügbar.",
+                "Es können mehrere Varianten pro Übung erstellt und für jeden Studierenden zufällig ausgewählt werden."
+            ]
+        },
+        "grading": {
+            "text": "<b>Benotung:</b> Dozenten können Notenschlüssel für Kurse und Prüfungen konfigurieren, um automatisch Noten zu berechnen und sie den Studierenden anzuzeigen. Die Noten können leicht als csv-Dateien exportiert werden, um sie in Universitätssysteme (wie Campus online) hochzuladen.",
+            "details": []
+        },
+        "assessment": {
+            "text": "<b>Bewertung:</b> Artemis verwendet eine Doppelblind-Bewertung und strukturierte Bewertungskriterien, um die Konsistenz und Fairness zu verbessern. Es integriert einen Bewertungs-Trainingsprozess (auf der Grundlage von Beispielaufgaben und vom Dozenten definierten Beispielbewertungen), verfügt über eine Bewertungs-Rangliste und ermöglicht den Studierenden, die Bewertungen zu bewerten. Die Studierenden können sich beschweren oder um mehr Feedback bitten.",
+            "details": []
+        },
+        "discussion": {
+            "text": "<a href=\"https://docs.artemis.ase.in.tum.de/user/discussion/\">Diskussion:</a> Dozenten können Ankündigungen veröffentlichen. Studierende können Fragen stellen, Kommentare abgeben und auf andere Beiträge reagieren. Tutoren können unbeantwortete Fragen filtern.",
+            "details": []
+        },
+        "notifications": {
+            "text": "<b>Benachrichtigungen:</b> Artemis unterstützt anpassbare Web- und E-Mail-Benachrichtigungen. Benutzer können verschiedene Benachrichtigungstypen aktivieren und deaktivieren.",
+            "details": []
+        },
+        "teamExercises": {
+            "text": "<b>Gruppenaufgaben:</b> Dozenten können Gruppenaufgaben mit Echtzeit-Zusammenarbeit und eigenen Tutoren pro Gruppe konfigurieren.",
+            "details": []
+        },
+        "lectures": {
+            "text": "<b>Vorlesungen:</b> Dozenten können Vorlesungsfolien hochladen, Vorlesungen in Einheiten unterteilen, Videostreams, Vorlesungsaufzeichnungen und Übungen in Vorlesungen integrieren und Lernziele definieren.",
+            "details": []
+        },
+        "integratedMarkdownEditor": {
+            "text": "<b>Integrierter Markdown-Editor:</b> Die Dozenten definieren Problemstellungen mit Markdown, um den Text einfach zu formatieren und Bilder und Links zu integrieren.",
```

```suggestion
            "text": "<b>Integrierter Markdown-Editor:</b> Die Dozenten können Problemstellungen im Markdown-Format definieren, um den Text einfach zu formatieren und Bilder und Links zu integrieren.",
```
The other points are worded with ‘Dozenten können…’